src/test/java/io/github/wimdeblauwe/hsbt/DummyApplication.java

```diff
@@ -2,14 +2,23 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+
+import io.github.wimdeblauwe.hsbt.mvc.PartialsController.TodoRepository;
 
 /**
  * Just created this here to make Spring Boot test slices work
  */
-@SpringBootApplication
+@SpringBootApplication(exclude = {SecurityAutoConfiguration.class}) // Security is on by default
 public class DummyApplication {
 
     public static void main(String[] args) {
         SpringApplication.run(DummyApplication.class);
     }
+
+    @Bean
+    TodoRepository repository() {
```

Comment:
is this needed?

Reply:
It is if you want to run the main method.

Reply:
It is not the intention to actually run this main. As the Javadoc states, it is only here to make the Spring Boot test slices work. Unless you know of a better alternative?

Reply:
I don't, but I find it useful to be able to run it, then I can debug a running app and see the results of hitting the endpoints.